### ISSUE TITLE
Change exp_parser and exp_saver to take opts Hash

### ISSUE THREAD

**Issue by esotericpig** [esotericpig/nurby#3]
Using an options Hash would probably be best for start_saver(...) and ExpSaver.new(...) readabiility.

And, it'd make it easier to maintain for future changes.

```Ruby
exp_parser.start_saver('v',{:stop_savers=>true,:only_if_no_saver=>true,:escape=>true,})
```

**Comment by esotericpig** [comment]
Possibly use keyword arguments:

```Ruby
def start_saver(id,stop_savers: true,escape: true,**options)
  local_variables.each do |symbol|
    options[symbol] = binding.local_variable_get(symbol) if symbol != :options
  end
  # ...
end

start_saver('id',extra: 51)
```

**Comment by esotericpig** [comment]
Fixed in commit bbcf442 in ExpParser, ExpSaver, etc.
